xrpl/asyncio/transaction/main.py

```diff
@@ -325,9 +408,8 @@ async def _calculate_fee_per_transaction_type(
 
     # Multi-signed Transaction
     # 10 drops × (1 + Number of Signatures Provided)
-    if transaction.signers and len(transaction.signers) > 0:
-        base_fee = net_fee * (1 + len(transaction.signers)) + base_fee
-
+    if signers_count is not None and signers_count > 0:
+        base_fee += net_fee * (1 + signers_count)
```

Comment:
Why do we need the `signers_count` parameter? (It seems like the same value is calculated automatically without passing that in)

(And if it's not needed in this function, can it also be removed from the function that calls this?)

Reply:
signers_count is the number of signers that a multisigned transaction expects to have. When this function is called in the multisign snippet, it hasn't yet been signed, so transaction.signers isn't correct. This autofills the transaction before the transaction is individually signed and multisigned. Also, this parameter isn't used in functions that call autofill (this function) because signers_count will only be used for multisigning and the other functions that call this function will never be used for multisigning.